definitions/tools/sequence_to_fastq_rna.cwl

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env cwl-runner
```

Comment:
Is there a reason that this is named `sequence_to_fastq_rna.cwl` instead of just `sequence_to_fastq.cwl`? Have I missed something that's RNA-specific here?